internal/perses/common/constants.go

```diff
@@ -16,8 +16,46 @@ limitations under the License.
 
 package common
 
+import "github.com/perses/perses-operator/api/v1alpha1"
+
 const (
 	PersesFinalizer     = "perses.dev/finalizer"
 	TypeAvailablePerses = "Available"
 	TypeDegradedPerses  = "Degraded"
+
+	// Volume names
+	ConfigVolumeName  = "config"
```

Comment:
actually is there a need to have these constants exported ?

From the usage, it seems it's only used in the common package and there is no need to export them. Excepting if you have an usage in mind, I would advise to keep it package private. It will be a bit cleaner